src/commcare_cloud/ansible/deploy_shared_dir.yml

```
@@ -1,2 +1,2 @@
 ---
-- import_playbook: roles/shared_dir.yml
+- import_playbook: deploy_efs_mount_shared_dir.yml
```

I'm not sure I understand why we have to replace the old playbook with the new one.

Keep in mind that third party environments use commcare-cloud too as an open source tool for managing their commcare deployments, and they won't be using EFS. So whatever we do we need to support both NFS and EFS.

---

A potential easy solution is to add

```suggestion
- import_playbook: roles/shared_dir.yml
  when: efs_shared_mount_dir is not defined
- import_playbook: deploy_efs_mount_shared_dir.yml
  when: efs_shared_mount_dir is defined
```

I could be wrong. If that makes sense to you, try running that against staging and making sure it does the right thing.